### DISCUSSION TITLE
Handling No Available Data Responses

### DISCUSSION THREAD

**Discussion by jeffkala** [nautobot/nautobot-app-chatops#105] [Q&A] [answered]
While reviewing the Meraki chatops plugin and a few other plugins wanted to start a discussion around the proper handling of a command that is "successful" but has no return data.  E.g. Ask for switches and there is none in the environment.  The command itself is successful, but there is no data to return.

Should the CommandStatusChoices.SUCCESS be used with a message of "No switches available".  Or should a standard markdown message be sent with "No switches available".  If neither whats the best option.  Looked through a few of the other chatops plugins and see a few of each of the above used.

**Comment by smk4664** [comment, marked as answer]
In this case, the command was still successful and should be logged in Nautobot. A nice friendly message should be sent to the end user. This is my opinion though.

**Reply by jeffkala** [reply]
I was sending `send_error` for these, but will update them and use STATUS_SUCCEEDED to make sure its logged in nautobot.

**Reply by jvanderaa** [reply]
I agree here as well. The command executed successfully.
